src/test/java/net/sf/jabref/logic/layout/format/LatexToUnicodeFormatterTest.java

```diff
@@ -28,9 +29,10 @@ public void testFormatStripLatexCommands() {
     @Test
     public void testFormatTextit() {
         // See #1464
-        assertEquals("text", formatter.format("\\textit{text}"));
+        assertEquals("\uD835\uDC61\uD835\uDC52\uD835\uDC65\uD835\uDC61", formatter.format("\\textit{text}"));
```

Comment:
It is not only the symbol for an invalid character, it is also the symbol for a Java escape (shown on the page you linked). I arrived at this sequence by copying the italicized text in Intellij into the code which transformed it into these hexes.

If you remove the first hex, then the first letter of the expected output turns into a `?` for Intellij, so this really seems to be the escape sequence.